installers/flash-tarball/radiasoft-download.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# To run: curl radia.run | bash -s flash-tarball
+#
+set -euo pipefail
+
+flash_main() {
+    local d="$PWD"
+    if [[ ! -f "$d/FLASH-4.6.2.tar.gz" ]]; then
+        install_err "$d/FLASH-4.6.2.tar.gz must exist"
+    fi
+    cd ~/src/radiasoft/rsflash
+    git checkout --quiet master
```

Comment:
Disagreeing with my comment below, I think we should:

```sh
install_url radiasoft/download installers
install_script_eval rpm-code/codes.sh
codes_download rsflash
```

Note that `install_url` changes the global state of `$install_url` so it's important to not do this too early, which you aren't. Note that `$codes_dir` will not be available in this use of codes, but that's ok (comment below stands).

Reply:
1d07dd5